### ISSUE TITLE
Question about naming of `lm_score` parameter in `decode_logits`

### ISSUE THREAD

**Issue by patrickvonplaten** [kensho-technologies/pyctcdecode#63]
Might not be super important, but I was a bit surprised by the naming of the `lm_score` variable in
https://github.com/kensho-technologies/pyctcdecode/blob/9071d5091387579b4722cfcbe0c8597ad0b16227/pyctcdecode/decoder.py#L498

the `lm_score` corresponds to the normalized sum of both the `logit_score` and the `lm_score` no? See: https://github.com/kensho-technologies/pyctcdecode/blob/9071d5091387579b4722cfcbe0c8597ad0b16227/pyctcdecode/decoder.py#L326

Shouldn't it rather be called `merged_score` or something?

**Comment by lopez86** [comment]
You're correct here. I added a small PR to clarify this.
